pkg/util/github/commit.go

```diff
@@ -0,0 +1,25 @@
+package github
+
+import (
+	"fmt"
+
+	"github.com/google/go-github/v42/github"
+
+	"github.com/merico-dev/stream/internal/pkg/log"
+)
+
+func (c *Client) GetLastCommit() (*github.RepositoryCommit, error) {
```

Comment:
Where is this function used?

Reply:
It's not used now. But it's ok left here as a util function at `util/github` package.